devito/finite_differences/finite_difference.py

```diff
@@ -232,12 +234,13 @@ def cross_derivative(expr, dims, fd_order, deriv_order, stagger=None, **kwargs):
 
     This is also more easily obtainable via:
 
-    >>> (f*g).dxdy
+    >>> (f*g).dxdy.evaluate
```

Comment:
now this is funny, this will randomly change as it looks like evaluation order is non deterministic so it will do x then y or y then x depending on the mood

Reply:
this is fixed right?

Reply:
yes